config/configuration.rb

```diff
@@ -0,0 +1,132 @@
+require 'pathname'
+require 'dotenv'
+
+# job composer app specific configuration
+class Configuration
+
+  # The app's configuration root directory
+  # @return [Pathname] path to configuration root
+  def config_root
+    Pathname.new(ENV["OOD_APP_CONFIG_ROOT"] || "/etc/ood/config/apps/myjobs")
+  end
+
+  def initializers_root
+    Pathname.new(ENV["OOD_APP_INITIALIZERS_ROOT"] || config_root.join("initializers"))
+  end
+
+  def load_external_config?
+    to_bool(ENV.fetch('OOD_LOAD_EXTERNAL_CONFIG', (rails_env == 'production')))
```

Comment:
More of a stylistic opinion, but this is also equivalent to:

```ruby
to_bool(ENV["OOD_LOAD_EXTERNAL_CONFIG"] || rails_env == "production)
```

So choose whichever you think is easiest to read.